openai-core/src/commonMain/kotlin/com.aallam.openai.api/run/RunStepDetails.kt

```diff
@@ -89,9 +89,18 @@ public sealed interface ToolCallStep {
     @SerialName("function")
     public data class FunctionTool(
         /**
-         * The name of the function.
+         * The ID of the tool call object.
          */
-        @SerialName("name") public val name: String,
+        @SerialName("id") public val id: String,
+        /**
+         * The definition of the function that was called.
+         */
+        @SerialName("function") public val function: FunctionDefinition,
+    ) : ToolCallStep
+
+    @BetaOpenAI
+    @Serializable
+    public data class FunctionDefinition (
```

Comment:
Could you please move this class out of `ToolCallStep` scope ? maybe we can name it `FunctionToolCallStep` or something like that :)